xbmc/addons/PVRClient.cpp

```diff
@@ -1269,7 +1268,6 @@ PVR_ERROR CPVRClient::OpenRecordedStream(const CFileItem& recordingItem)
   }
 
   return DoAddonCall(__FUNCTION__, [this, recording](const AddonInstance* addon) {
```

Comment:
> xbmc/addons/PVRClient.cpp:1270:37: warning: lambda capture 'this' is not used [-Wunused-lambda-capture]
```suggestion
   return DoAddonCall(__FUNCTION__, [recording](const AddonInstance* addon) {
```

Reply:
I had tried that in response to the Jenkins log, but the [this] capture is needed in those functions.  The module will not compile if you remove them.  Perhaps some difference across the various compilers here.

Reply:
You only removed it in this lambda capture or also in the other one?
As the compiler warning indicates it's only unused here. If that still gives errors please tell which compiler you are using and post the compile error you get.